core/core-rest/RestClient.cpp

```diff
@@ -1485,7 +1485,7 @@ void RestClient::listFriends(
 
         if (limit) args.emplace("limit", std::to_string(*limit));
         if (state) args.emplace("state", std::to_string((int32_t)*state));
-        if (!cursor.empty()) args.emplace("cursor", cursor);
+        if (!cursor.empty()) args.emplace("cursor", encodeURIComponent(cursor));
```

Comment:
I don't understand why these cursor diffs are appearing. I think it's a github bug? It's identical to what I merged here? https://github.com/heroiclabs/nakama-cpp/pull/121/files#diff-05392393d7b85e7af2aebb95d2100ea8684ea007b840a0db47d5e75ee4306a5bR1488

Reply:
fixed